tsl/src/nodes/decompress_chunk/decompress_chunk.h

```diff
@@ -49,6 +49,13 @@ typedef struct DecompressChunkPath
 	 * uncompressed chunk, but are still used for decompression.
 	 */
 	List *decompression_map;
+
+	/*
+	 * Whether a targetlist entry is a segmentby column. Parallel to the above
```

Comment:
It took me some time to understand what this list actually contains. Could you expand the comment a bit and mention that it has the same length as the list above and contains int values for each attribute to indicate if it is a `segment_by` column or not?

Reply:
I changed the comment, does it look better? I have trouble formulating this in a comprehensible way...